frontend/apps/reader/modules/readerlink.lua

```diff
@@ -316,6 +316,153 @@ local function isSwipeToJumpToLatestBookmarkEnabled()
     return G_reader_settings:isTrue("swipe_to_jump_to_latest_bookmark")
 end
 
+function ReaderLink:getFootnoteSettingsMenuTable()
+    local menu_items = {
+        {
+            text = _("Show footnotes in popup"),
+            enabled_func = function()
+                return isTapToFollowLinksOn() or isSwipeToFollowNearestLinkEnabled()
```

Comment:
All this was moved in this function. Hard to see if there were changes, so:
Only a move ? Or did you modify some stuff?